lib/screens/episode.dart

```diff
@@ -236,10 +238,16 @@ class _EpisodeScreenState extends ConsumerState<EpisodeScreen> with AutoRouteAwa
         onSelectionChanged: (id) {
           var episodeUrl = 'https://brunstad.tv/episode/${widget.episodeId}';
           if (id == 'fromStart') {
-            Share.share(episodeUrl);
+            SharePlugin.Share.share(episodeUrl);
           } else {
-            Share.share('$episodeUrl?t=$currentPosSeconds');
+            SharePlugin.Share.share('$episodeUrl?t=$currentPosSeconds');
           }
+          ref.read(analyticsProvider).shared(Share(
+                pageName: 'episode',
```

Comment:
```suggestion
                pageCode: 'episode',
```
I updated the ADR too